modules/text2vec-openai/clients/vectorizer.go

```diff
@@ -50,41 +51,46 @@ type openAIApiError struct {
 type vectorizer struct {
 	apiKey     string
 	httpClient *http.Client
-	urlBuilder *openAIUrlBuilder
+	host       string
+	path       string
 	logger     logrus.FieldLogger
 }
 
 func New(apiKey string, logger logrus.FieldLogger) *vectorizer {
 	return &vectorizer{
 		apiKey:     apiKey,
 		httpClient: &http.Client{},
-		urlBuilder: newOpenAIUrlBuilder(),
+		host:       "https://api.openai.com",
+		path:       "/v1/embeddings",
 		logger:     logger,
 	}
 }
 
 func (v *vectorizer) Vectorize(ctx context.Context, input string,
 	config ent.VectorizationConfig,
 ) (*ent.VectorizationResult, error) {
-	return v.vectorize(ctx, input, v.docUrl(config.Type, config.Model))
+	return v.vectorize(ctx, input, v.getModelString(config.Type, config.Model, "document"))
 }
 
 func (v *vectorizer) VectorizeQuery(ctx context.Context, input string,
 	config ent.VectorizationConfig,
 ) (*ent.VectorizationResult, error) {
-	return v.vectorize(ctx, input, v.queryUrl(config.Type, config.Model))
+	return v.vectorize(ctx, input, v.getModelString(config.Type, config.Model, "query"))
 }
 
 func (v *vectorizer) vectorize(ctx context.Context, input string,
-	url string,
+	model string,
 ) (*ent.VectorizationResult, error) {
 	body, err := json.Marshal(embeddingsRequest{
 		Input: input,
+		Model: model,
 	})
 	if err != nil {
 		return nil, errors.Wrapf(err, "marshal body")
 	}
 
+	url := v.host + v.path
```

Comment:
just a suggestion, you could use `url.JoinPath()` method here

Reply:
confirmed by @antas-marcin that it's fine to require 1.19

Reply:
done